components/automate-chef-io/content/docs/iam-v2-overview.md

```diff
@@ -80,13 +80,14 @@ A policy statement must include a `Role`, or a list of inline `Actions`.
 
 Property   | Description
 -----------|-----------------------------
-Role       | a named list of actions
+Role       | an alias for a list of actions
+Actions    | an inline list of actions
```

Comment:
PR Bonus: `Actions` was missing. Added that and revised `Role` to (I hope) show clearly the difference between the two.

Reply:
@msorens i thought we had agreed as a team to keep actions out of everything but a mention in the api reference as a way to encourage folks to use roles and only actions in special circumstances... has that changed?

Reply:
@msorens I removed Actions from this chart since its inclusion could be potentially confusing. Users might get confused and include a role AND a list of actions, when only one of them is allowed in a policy statement.

Reply:
Had forgotten that @susanev, thanks. So we're good with @mjingle 's revert of it then.